src/app/identity.service.ts

```diff
@@ -35,6 +36,13 @@ export class IdentityService {
     );
   }
 
+  getResetPasswordToken(email: string) {
```

Comment:
Would it make things clearer if we make the return type explicit (yes, typing is one of my favourite things, but I'm not enough of a TS expert yet to be confident about where it's better to be explicit and where it's better to rely on inference)

```suggestion
  getResetPasswordToken(email: string): Observable<{jwt: string}> {
```

Reply:
I've changed it to `getResetPasswordToken(email: string): Observable<[]>` because the endpoint returns an empty list